### PR TITLE
Test: More usages of Parameters/ForwardCtor to make it DRYer

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

File: source/agora/test/LocalTransactions.d [hunk 1]
```diff
@@ -146,13 +124,12 @@ unittest
 
     static class PickyValidator : TestValidatorNode
     {
-        public this (Config config, Registry* reg, immutable(Block)[] blocks,
-            in TestConf test_conf, shared(time_t)* cur_time)
+        public this (Parameters!(TestValidatorNode.__ctor) args)
```
Comment (hewison-chris) [inline source/agora/test/LocalTransactions.d]:
```suggestion
        public this (Parameters!(typeof(super).__ctor) args)
```

**Review by hewison-chris — APPROVED** [review]
Only one consistency sugestion.

File: source/agora/test/LocalTransactions.d (hunk 1 above)
Comment (Geod24) [reply source/agora/test/LocalTransactions.d]: Done

**Review by kchulj — APPROVED** [review]
Nice usage of `template mixin`.

**Comment by linked0** [comment]
In my local machine, I got errors like this. Could you tell me what I should do for that error?

```
Linking...
Undefined symbols for architecture x86_64:
  "_main", referenced from:
     implicit entry/start for main executable
     (maybe you meant: __D2rt6dmain212_d_run_main2UAAamPUQgZiZ6runAllMFZ9__lambda1MFZv, __D2rt6dmain212_d_run_main2UAAamPUQgZiZ7tryExecMFMDFZvZv , __d_run_main2 , __D2rt6dmain212_d_run_main2UAAamPUQgZiZ6runAllMFZv , __D4core6thread10threadbase10ThreadBase7sm_mainCQBtQBrQBnQBe , __D4core6thread8osthread16_mainThreadStoreG312v , __D6geod249LocalRest14is_main_threadb , __d_run_main )
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Error: /usr/bin/cc failed with status: 1
ldc2 failed with exit code 1.
```

**Pull request merged**